internal/librarian/generate_test.go

```diff
@@ -16,14 +16,16 @@ package librarian
 
 import (
 	"context"
-	"github.com/googleapis/librarian/internal/docker"
 	"os"
 	"os/exec"
 	"path/filepath"
+	"regexp"
+	"sort"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/googleapis/librarian/internal/config"
+	"github.com/googleapis/librarian/internal/docker"
```

Comment:
Could you remove these unused import?

Reply:
This import is used. The `ContainerClient` interface has methods require types from docker package
https://github.com/googleapis/librarian/blob/07e9327ba01e9fced917b686b36e546614f4361d/internal/librarian/generate_test.go#L33-L37